tests/monitors/supervisor/supervisor_test.py

```diff
@@ -0,0 +1,38 @@
+from contextlib import contextmanager
+from functools import partial as p
+
+import pytest
+from tests.helpers.agent import Agent
+from tests.helpers.assertions import has_datapoint_with_dim, tcp_socket_open
+from tests.helpers.metadata import Metadata
+from tests.helpers.util import container_ip, run_service, wait_for
+from tests.helpers.verify import verify
+
+pytestmark = [pytest.mark.monitor_with_endpoints]
+
+
+METADATA = Metadata.from_package("supervisor")
+PORT = 9001
+PROCESS = "long_script"
+
+
+@contextmanager
+def run_php_fpm():
```

Comment:
Shouldn't this be `run_supervisor`?  Same for a few things below.

Reply:
oh good catch sorry ^^ fixed